code/game/objects/structures/beds_chairs/bed.dm

```diff
@@ -162,6 +164,7 @@
 	buildstacktype = /obj/item/stack/sheet/mineral/wood
 	buildstackamount = 10
 	var/mob/living/owner = null
+	comfort = 0.5
```

Comment:
Good, catmemes can sleep on the floor for all i care.

It is of critical importance we give Ian some AI to regain health by sleeping in a dog bed.